PanKanapka/clients/models.py

```diff
@@ -1,7 +1,5 @@
 from django.db import models
-from django.contrib.auth.models import (
-    BaseUserManager, AbstractBaseUser
-)
+from django.contrib.auth.models import BaseUserManager, AbstractBaseUser, User
```

Comment:
Czemu importujemy tutaj `User`? Domyślna klasa usera jest nadpisana i z niej korzystamy przy składaniu zamówień (zakładam, że też przy rejestracji i logowaniu?).

Reply:
To też nigdzie nie jest wykorzystane :) , już usunięte